MCCP/mccp-23.md

```diff
@@ -0,0 +1,71 @@
+---
+mccp: 23
+title: Disable underutilized dials II
+status: Proposed
+author: Dimitri Golecko (@dimsome), Julian Kusnetzoff (@jkusne)
+discussions-to: https://forum.mstable.org/t/rfc-disable-all-dials-except-staking-treasury-and-create-a-v2-incentives-one/894
```

Comment:
Can leave blank for now as this should point to the forum discussion when we post this MCCP, will update later

Reply:
done